lib/ansible/plugins/doc_fragments/acme.py

```diff
@@ -85,7 +90,6 @@ class ModuleDocFragment(object):
          (staging and production) and against the
          L(Pebble testing server,https://github.com/letsencrypt/Pebble)."
     type: str
-    default: https://acme-staging.api.letsencrypt.org/directory
```

Comment:
I'd leave this in and remove the duplicate from the `description` section. That's the standard way we document default values, and having `default:` as a field in the docs makes them easier to scan.

Reply:
After discussion on the #ansible-community IRC channel, I understand better what is happening in this PR. Users will currently get a warning that they're relying on a default value that's going to go away.